config/environments/development.rb

```diff
@@ -30,6 +30,7 @@
 
   # Store uploaded files on the local file system (see config/storage.yml for options)
   config.active_storage.service = :local
+  config.active_storage.content_types_to_serve_as_binary -= ['image/svg+xml']
```

Comment:
maybe drop a comment about how we are explicitly trusting this because are going to be referencing these from `<img>` tags which don't execute scripts?